packages/calcite-components/src/components/input-number/input-number.e2e.ts

```diff
@@ -554,20 +554,36 @@ describe("calcite-input-number", () => {
       await input.callMethod("setFocus");
       await page.waitForChanges();
 
+      const eventSpy = await page.spyOnEvent("keydown");
       await page.keyboard.down("ArrowUp");
       await page.waitForTimeout(delayFor2UpdatesInMs);
       await page.waitForEvent("calciteInputNumberInput");
+
+      expect(eventSpy).toHaveReceivedEventTimes(1);
+      expect(eventSpy.lastEvent.defaultPrevented).toBe(true);
```

Comment:
Side note: we should look into updating other tests to assert for event cancelation. Might be worth exploring a util for this. Otherwise, that's gonna be a lot of `expect`s. 🥲

Reply:
Agreed. A common util would be helpful here